topic-operator/pom.xml

```diff
@@ -135,11 +135,6 @@
             <classifier>test</classifier>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-migrationsupport</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
just out of curiosity - it was some old leftover and it wasn't used at all?

Reply:
It was needed for the EmbeddedKafkaCluster in Integration tests. But they improved it a bit in Kafka 3.0 so it is not needed anymore.